wp-parsely.php

```diff
@@ -179,11 +174,20 @@ public static function get_configs() {
 	 * Get Parse.ly options.
 	 */
 	public static function get_parsely_options(): array {
-		if ( ! isset( self::$parsely_options ) ) {
-			self::$parsely_options = get_option( 'parsely', [] );
+		if ( ! self::is_active() ) {
+			return array();
 		}
 
-		return is_array( self::$parsely_options ) ? self::$parsely_options : [];
+		$parsely = new Parsely();
+
+		/**
+		 * Parse.ly options.
+		 *
+		 * @var array
+		 */
+		$parsely_options = $parsely->get_options();
+
+		return $parsely_options;
```

Comment:
I think we can just use `$_GLOBALS['parsely']`?

```suggestion
		/**
		 * Parse.ly options.
		 *
		 * @var array
		 */
		$parsely_options = array();
		if ( isset( $GLOBALS['parsely'] ) && is_a( $GLOBALS['parsely'], 'Parsely\Parsely' ) ) {
			$parsely_options = $parsely->get_options();
		}
		
		return $parsely_options;
```

Reply:
Addressed in https://github.com/Automattic/vip-go-mu-plugins/pull/5051/commits/053a7d6895695604ad842bf72e735ebdb13917a1